src/main/scala/Main.scala

```diff
@@ -53,10 +59,15 @@ object Main {
 
   def main(args: Array[String]): Unit = {
 
+
     type ErrString = String
 
     parser.parse(args, Config(null)) match {
       case Some(conf) => {
+        val myFormatter: Formatter = formatter"[$levelColored] $message$newLine"
```

Comment:
Move the logger setup code to either (1) start of main, or (2) a separate function in Main.scala.